10-JavaScript/js-intro.md

```diff
@@ -0,0 +1,439 @@
+- `undefined`
+- `Object`
+- `Function`
+  - similar to arrays, technically these aren't a distinct type, but we won't get into that now, and it's worthwhile to separate it on this list
+
+The best way for us to get into JavaScript at the moment is to blitz through all of the familiar types. Feel free to follow along in the Node REPL to verify that these these are real things in JavaScript.
+
+#### An Example Error in Node REPL
+
+If something is not real, Node REPL will let us know by _throwing an error._
+
+For example, if we have the Node REPL open and type in something that the Node REPL doesn't recognize, it may have output that looks like any of these options:
+
+```
+> asdf
+Thrown:
+ReferenceError: asdf is not defined
+> this isn't real
+Thrown:
+this isn't real
+     ^^^
+
+SyntaxError: Unexpected identifier
+```
+
+### Boolean means `true` or `false`
+
+Check in Node REPL that you can evaluate a boolean `true` or `false` and get back what you'd expect, with similar syntax.
+
+In your Node REPL, input these lines. Do you get what you expect?
+
+1. `true;`
+1. `false;`
+1. `true && true;`
+1. `true && false;`
+1. `false || true;`
+1. `false || false;`
+
+<details>
+
+  <summary>
+    Compare your answers here
+  </summary>
+
+  1. `true`
+  1. `false`
+  1. `true`
+  1. `false`
+  1. `true`
+  1. `false`
+
+</details>
+
+### `Number` represents a Numeric Value
+
+In JavaScript, the `Number` type includes support for the following kinds of numbers:
+
+- integers (1, 2, 3, etc.)
+- floats (1.4, -40.1)
+- infinity (+Infinity, -Infinity)
+- `NaN` which means "not a number." `NaN` is returned when you do a numeric operation on anything that's not a `Number`.
+
+In your Node REPL, input these lines. Do you get what you expect?
+
+1. `4;`
+1. `2.0;`
+1. `4 / 2;`
+1. `4 / 2.0;`
+1. `4.0 + 2.0`
+1. `4 + 2.1`
+1. `Infinity;`
+1. `+Infinity;`
+1. `-Infinity;`
+1. `Number.MAX_VALUE;`
+1. `Infinity > Number.MAX_VALUE;`
+1. `0.1 + 0.2 == 0.3;`
+1. `0.1 + 0.2;`
+1. `'asdf' - 5;`
+
+<details>
+
+  <summary>
+    Compare your answers here
+  </summary>
+
+  1. `4`
+  1. `2`
+  1. `2`
+  1. `2`
+  1. `6`
+  1. `6.1`
+  1. `Infinity`
+  1. `Infinity`
+  1. `-Infinity`
+  1. `1.7976931348623157e+308`
+  1. `true`
+  1. `false`
+  1. `0.30000000000000004`
+  1. `NaN`
+
+  We will not spend the time to go into the specifics and the "why"s of JavaScript at the moment; for now, be satisfied with the mystery, curiosity, and difference you can see.
+
+</details>
+
+### `Strings` are declared with `""`, `''`, or backticks `    ``    `.
+
+The idea of Strings is the same between JavaScript and Ruby. However, these are the following differences we will highlight:
+
+- In JavaScript, strings declared with double-quotes and strings declared with single-quotes are the same
+- In those cases, we will standardize on using single-quotes `''`
+- **We can only use backticks for string interpolation**
+- The syntax for string interpolation is with `${}`
```

Comment:
I might change the bolded line to something like

- Neither single- nor double-quoted strings allow string interpolation. If you want to interpolate, you must use backticks (` `` `).